Forward port c6ef88ba01cc47ac4c6a2cfe51e15eaa4d833476 for the 5.14 series
--- a/include/net/inet_connection_sock.h.orig	2021-08-29 15:04:50.000000000 -0700
+++ b/include/net/inet_connection_sock.h	2021-09-20 15:19:51.125722417 -0700
@@ -134,8 +134,8 @@ struct inet_connection_sock {
 	u32			  icsk_probes_tstamp;
 	u32			  icsk_user_timeout;
 
-	u64			  icsk_ca_priv[104 / sizeof(u64)];
-#define ICSK_CA_PRIV_SIZE	  sizeof_field(struct inet_connection_sock, icsk_ca_priv)
+#define ICSK_CA_PRIV_SIZE	  (216)
+	u64			  icsk_ca_priv[ICSK_CA_PRIV_SIZE / sizeof(u64)];
 };
 
 #define ICSK_TIME_RETRANS	1	/* Retransmit timer */
